plugins/modules/configure_network_settings.py

```diff
@@ -127,13 +149,18 @@ def main():
     retries = module.params["retries"]
     delay = module.params["delay"]
     vxlanPort = module.params["vxlanPort"]
+    routing_via_host = module.params["routing_via_host"]
+    ip_forwarding = module.params["ip_forwarding"]
 
     # Ensure patching is needed
-    if not any([mtu, geneve_port, ipv4_subnet, vxlanPort]):
+    if not any([mtu, geneve_port, ipv4_subnet, vxlanPort, routing_via_host, ip_forwarding]):
```

Comment:
@miheer as we discuss in slack, it should be ..., right?
`if not any([mtu, geneve_port, ipv4_subnet, vxlanPort, routing_via_host, ip_forwarding]):`
```suggestion
    if not any([mtu, geneve_port, ipv4_subnet, vxlanPort, routing_via_host, ip_forwarding, network_type]):
```